index.js

```diff
@@ -188,5 +188,12 @@ const saveEditChanges = (event) => {
 
 };
 
+export const valueCheck = (num1, num2) => {
+  if (num1 > num2) {
+    return "Num1 is Bigger then Num2";
+  }
+  return "Num1 is Smaller then Num2";
+}
```

Comment:
**Suggestion:** The function `valueCheck` could be more generic if it returned a boolean value instead of a string. This would allow it to be used in more contexts. [enhancement]
```suggestion
export const isNum1GreaterThanNum2 = (num1, num2) => {
  return num1 > num2;
}
```